From 8043907d74827537109a782d4813a6e4a7a8dfbb Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Mon, 11 Apr 2016 05:40:06 -0400
Subject: [PATCH] pyanaconda/flags.py: drop selinux module

And we do not use selinux module for wrlinux, so drop it.

The fix is to revert upstream commit
...
commit b359e1a011ca71e5431969ae4f12bb847017eaa9
Author: Chris Lumens <clumens@redhat.com>
Date:   Wed Aug 10 13:08:26 2011 -0400

    Remove as many of the /selinux path hardcodings as possible (#729563).
...

Upstream-Status: Inappropriate <wrlinux specific>

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 pyanaconda/flags.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/pyanaconda/flags.py b/pyanaconda/flags.py
index ac774ed..8ba62d9 100644
--- a/pyanaconda/flags.py
+++ b/pyanaconda/flags.py
@@ -17,7 +17,6 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 
-import selinux
 import shlex
 import glob
 from pyanaconda.constants import SELINUX_DEFAULT, CMDLINE_APPEND
@@ -86,7 +85,7 @@ class Flags(object):
                   "nombr", "gpt"):
             self.set_cmdline_bool(f)
 
-        if not selinux.is_selinux_enabled():
+        if not os.path.exists("/selinux/load"):
             self.selinux = 0
 
 cmdline_files = ['/proc/cmdline', '/run/install/cmdline',
-- 
1.9.1

